bigtop-packages/src/common/logstash/do-component-build

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+. `dirname $0`/bigtop.bom
+
+export RELEASE=1
+export LANG="en_US.UTF-8"
+export JAVA_HOME=`readlink -f /usr/bin/java | sed "s:jre/bin/java::"`
```

Comment:
bigtop have a script to detect java home. Maybe we can leverage it.
```
# Autodetect JAVA_HOME if not defined
. /usr/lib/bigtop-utils/bigtop-detect-javahome
```

Reply:
Agree to leverage Bigtop script to detect JAVA_HOME, thanks for the info.
But I found that it's unnecessary to detect JAVA_HOME in `do-component-build` , and in the phase of `do-component-build` , `bigtop-utils` has not be ready yet.
Removed the JAVA_HOME detection.
Thanks for the comments, envans!

Reply:
Got it thanks for checking it :)